gcloud-java-dns/src/main/java/com/google/cloud/dns/testing/LocalDnsHelper.java

```diff
@@ -138,7 +154,8 @@
     ZONE_GET("GET", CONTEXT + "/[^/]+/managedZones/[^/]+"),
     ZONE_LIST("GET", CONTEXT + "/[^/]+/managedZones"),
     PROJECT_GET("GET", CONTEXT + "/[^/]+"),
-    RECORD_LIST("GET", CONTEXT + "/[^/]+/managedZones/[^/]+/rrsets");
+    RECORD_LIST("GET", CONTEXT + "/[^/]+/managedZones/[^/]+/rrsets"),
+    BATCH("POST", "/batch");
```

Comment:
Can you be more specific in what you have in mind? We do have integration tests for batch which are using the same RPC components.